core/state/statedb_evm.go

```diff
@@ -64,13 +64,12 @@ func (s *Statedb) GetCodeSize(address common.Address) int {
 
 // AddRefund refunds the specified gas value
 func (s *Statedb) AddRefund(gas uint64) {
-	// @todo
+	s.refund += gas
```

Comment:
gas can be zero?

Reply:
EVM use this API to refund gas, as for the gas value, generally > 0.